server/public/styles.css

```diff
@@ -0,0 +1,173 @@
+.hidden {
+  display: none !important;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  min-height: 680px;
+  max-width: 900px;
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+  font-size: 16px;
+}
+
+main {
+  height: 86.5vh;
+  padding: 0.5rem 0.75rem;
+}
+
+h2 {
+  font-family: 'Signika', Helvetica, sans-serif;
+  font-weight: 700;
+  text-align: center;
```

Comment:
Bootstrap has text helpers you should use in most situations.

https://getbootstrap.com/docs/4.5/utilities/text/